src/gtimelog/timelog.py

```diff
@@ -361,6 +364,26 @@ def totals(self, tag=None, filter_text=None):
                 total_work += duration
         return total_work, total_slacking
 
+    @classmethod
+    def _get_grouped_order_key(cls, sorted_by, sorted_tasks):
+        """
+        Returns a function suitable as key parameter for the ordered function
```

Comment:
I'm not sure what "for the ordered function" refers to.

```suggestion
        """Return a callable usable as the `key` argument of sorted().
```

Reply:
Done.